.reuse/dep5

```diff
@@ -3,3 +3,7 @@ Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Files: .github/*
 Copyright: 2020 tqtezos
 License: MIT
+
+Files: nix/*
+Copyright: Nicolas Mattia <https://nmattia.com>
```

Comment:
Minor, but AFAIU it should apply only to `sources.nix`, but not `sources.json`. Even if `sources.json` is auto-generated, it's generated from our data, so it's more appropriate to assign copyright to tqtezos.

Reply:
Fixed.